CRM/Core/Error.php

```diff
@@ -586,6 +586,9 @@ public static function debug_log_message($message, $out = FALSE, $prefix = '', $
       if ($config->userSystem->is_drupal and function_exists('watchdog')) {
         watchdog('civicrm', '%message', ['%message' => $message], $priority ?? WATCHDOG_DEBUG);
       }
+      elseif ($config->userSystem->is_drupal and CIVICRM_UF == 'Drupal8') {
+        \Drupal::logger('civicrm')->log($priority ?? \Drupal\Core\Logger\RfcLogLevel::DEBUG, '%message', ['%message' => $message]);
```

Comment:
@sunilpawar @demeritcowboy:

(1) Are y'all sure that the pass-thru of `$priority` is right here? The thing is that PSR-3 and PEAR have different constants for log-levels/log-priorities. (PSR-3 uses strings; PEAR uses ints.) Observe that:

* In `CRM_Core_Error_Log`, it uses an array-map to ensure that it gives a PEAR-style value.
* Line 568 above passes $priority into a PEAR logger... so it had better be a PEAR-style value.
* The docblock therefore seems wrong - ie `$prioritity` is numeric. (The docblock in`Log_file::log()` is *also* wrong... but if you drill-down to `Log_file::log()`  => `Log::_format()` => `Log::priorityToString()`, it really does expect PEAR's numeric constants.)

I'm pretty sure this would need to reverse the mapping.

(2) There is a pre-existing issue (not the fault of this PR, but relevant to note) that the relationship between `CRM_Core_Error_Log` (Civi's PSR-3 stub) and `CRM_Core_Error::debug_log_message()` (Civi's PEAR wrapper) is bit off - IMHO, it's backwards. The relevance is that the current chain (CRM_Core_Error_Log => CRM_Core_Error => D8 Log) produces lossy a conversion. Without the structure of `$message` and `$context`, it'll be impossible to have something in the D8 realm to Nice Things (like pretty-printing, categorizing, translating). If the chain were setup better, then it wouldn't lose anything.

(Again, that's not the fault of this PR...)